src/components/experiment-tracking/tooltip/tooltip.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import './tooltip.css';
+
+export const Tooltip = ({ content, visible, pos, direction }) => {
```

Comment:
maybe we can name it something else rather than Tooltip? As it only uses for the Metrics chart in the experiment tracking. Any suggestions are welcome :)

Reply:
Maybe `MetricsChartsToolip`?